buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -23,18 +23,19 @@ import org.elasticsearch.gradle.test.RestIntegTestTask
 import org.elasticsearch.gradle.test.RunTask
 import org.gradle.api.Project
 import org.gradle.api.Task
+import org.gradle.api.tasks.SourceSet
 import org.gradle.api.tasks.bundling.Zip
 
 /**
  * Encapsulates build configuration for an Elasticsearch plugin.
  */
-class PluginBuildPlugin extends BuildPlugin {
+public class PluginBuildPlugin extends BuildPlugin {
```

Comment:
FWIW this was feedback from me, mainly around two things:
* use of explicit types (instead of `def`) helps readability because its easier to see what objects are being worked with: we should just opt into type safety.
* use of explicit access modifiers (instead of all implicitly `public`) helps readability because its easier to see what is implementation detail vs "api": we should just opt into proper visibility.